integration/test/metric/query-json.json

```diff
@@ -1,23 +0,0 @@
-[
-  {
-    "Id": "m1",
-    "MetricStat": {
-      "Metric": {
-        "Namespace": "MetricValueBenchmarkTest",
```

Comment:
Mind if I re-add something like this later with a readme for anyone(or future me) who needs to iterate fast on how to query their metric for test? I used this to call cw from my devdesktop because fetching wasn't working

Reply:
Yeah no problem. I just saw it wasn't being used anywhere.